app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()

```

Comment:
The `NotVaccinatedError` should be raised with a descriptive message. Consider using: `raise NotVaccinatedError('Visitor is not vaccinated')`.